package.json

```diff
@@ -13,12 +13,13 @@
   "dependencies": {
     "aws-sdk": "^2.1090.0",
     "axios": "^0.26.1",
-    "dotenv": "^16.0.0",
```

Comment:
we don't need this since we are always passing env vars either via `docker-compose` or CloudFormation